pkg/yttlibrary/assert.go

```diff
@@ -25,6 +26,27 @@ var (
 
 type assertModule struct{}
 
+func (b assertModule) Equals(thread *starlark.Thread, f *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	if args.Len() != 2 {
+		return starlark.None, fmt.Errorf("expected two arguments")
+	}
+
+	expected, err := core.NewStarlarkValue(args.Index(0)).AsGoValue()
+	if err != nil {
+		return starlark.None, err
+	}
+	actual, err := core.NewStarlarkValue(args.Index(1)).AsGoValue()
+	if err != nil {
+		return starlark.None, err
+	}
+
+	if expected != actual {
+		return starlark.None, fmt.Errorf("unequal: %s is not equal to the expected value %s", actual, expected)
```

Comment:
For `actual` and `expected` here, since the formatting for `int64`, `float64`, etc do not have easily readable string representations and we do not yet have a helper function available for this, we can add a little converter. Something like this that I whipped up below:

```golang
func scalarToString(val interface{}) string {
	switch typedVal := val.(type) {
	case float64:
		return strconv.FormatFloat(typedVal, 'f', -1, 64)
	case int64:
		return strconv.FormatInt(typedVal, 10)
	case bool:
		return strconv.FormatBool(typedVal)
	case nil:
		return "nil"
	}
	return fmt.Sprintf("%s", val)
}
```